terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -25,6 +25,7 @@
 from apps.project_management.graphql.projects import ProjectNode
 from apps.project_management.models import Project, Site, sites
 from apps.soil_id.models import SoilData
+from apps.soil_id.models.project_soil_settings import ProjectSoilSettings
```

Comment:
issue: this import causes a linter error. should we remove it, or do we need to make a change to use it?